internal/monitors/device-updater/device_updater.go

```diff
@@ -24,12 +27,12 @@ type DeviceUpdater struct {
 	devStore   service.DeviceStore
 }
 
-func NewDeviceUpdater(log logrus.FieldLogger, db *gorm.DB, store *store.Store) *DeviceUpdater {
+func NewDeviceUpdater(log logrus.FieldLogger, db *gorm.DB, stores *store.Store) *DeviceUpdater {
 	return &DeviceUpdater{
 		log:        log,
 		db:         db,
```

Comment:
just a general question but if we are abstracting the db as store do we also need db here? I don't see it used.